lexik/jwt-authentication-bundle/2.3/manifest.json

```diff
@@ -5,6 +5,7 @@
     "copy-from-recipe": {
         "etc/": "%ETC_DIR%"
     },
+    "aliases": ["jwt-auth"],
```

Comment:
If it's decided that this bundle is "the JWT solution for Symfony", then we could even go for adding `jwt` too as an alias.